app/webpacker/css/darkswarm/lists.scss

```diff
@@ -1,7 +1,6 @@
 body ol {
   list-style-type: none;
   margin-left: 0em;
-  padding-top: 1em;
```

Comment:
This could affect more than the Trix editor.

```
git grep -w ol
...
app/views/admin/order_cycles/_coordinator_fees.html.haml:%ol.coordinator-fees
app/views/admin/order_cycles/_exchange_form.html.haml:    %ol{ ng: { show: 'enterprises[exchange.enterprise_id].managed || order_cycle.viewing_as_coordinator' } }
```

Did you check if the order cycle editing still looks the same?

Reply:
@mkllnk I've checked the order cycle editing and it still looks the same I've also checked other areas and I haven't found anything looking odd.